.github/workflows/build_md.yml

```diff
@@ -0,0 +1,47 @@
+name: Markdown Build
+on:
+  pull_request:
+    branches:
+      - master
```

Comment:
Should this run on every pull request? Seems more like a manual run thing as well, like the release is.

Reply:
My rationale is that we'd like to know whether it's able to convert to markdown, but it does use very much the same code as a FGD compile so I suppose we don't need to run it outside of releases.

Reply:
Switched to over to be apart of the release CI instead of build.